hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,116 +24,112 @@
       tests: ["flash_ctrl_smoke"]
     }
     {
-      name: backpressure
+      name: smoke_sw_op
       desc: '''
-            Sanity + randomly pick between back-pressured reads / writes and fifo status-polled
-            reads / writes via the controller.
+            Perform flash protocol controller read, program and erase on the single page of one 
+            bank within Data partition.
             '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: all_partitions
-      desc: '''
-            Sanity + both, legal data and info partitions are accessed. In future, support for
-            multiple info partitions may be added - those will be covered as well.
-            '''
-      milestone: V2
-      tests: ["flash_ctrl_rand_ops"]
+      milestone: V1
+      tests: ["flash_ctrl_smoke_sw_op"]
     }
     {
-      name: host_read_check
+      name: smoke_hw_rd
       desc: '''
-            Sanity + the success of each operation is verified by randomly choosing
-            between backdoor and via the host interface.
+            Perform host direct read on the single page of one bank within
+            Data partition.
             '''
-      milestone: V2
-      tests: []
+      milestone: V1
+      tests: ["flash_ctrl_smoke_hw_rd"]
     }
     {
-      name: host_illegal_access
+      name: host_ctrl_hw_if
       desc: '''
-            Do a bunch of illegal accesses via host interface to ensure that an error response is
-            appropriately seen. This includes writes, out of bound reads, protocol violated reads
-            and all other unsupported accesses.
+            Perform read of seed materials via Hardware Interface of the Flash Protocol Controller.
```

Comment:
will we also check that during rma entry the contents of flash are correctly wiped out?

Reply:
This is test for HW IF datapath including wiping out Creator, Owner, Isolated and all data partitions.

Reply:
sounds good.

Reply:
Added "During RMA entry, check if content of flash is wiped out."